utilities_common/sfp_helper.py

```diff
@@ -47,7 +43,15 @@
     'e2_server_firmware': 'E2 Server Firmware'
 }
 
+C_CMIS_DELTA_DATA_MAP = {
+    'supported_max_tx_power': 'Supported Max TX Power',
+    'supported_min_tx_power': 'Supported Min TX Power',
+    'supported_max_laser_freq': 'Supported Max Laser Frequency',
+    'supported_min_laser_freq': 'Supported Min Laser Frequency',
+}
+
```

Comment:
@mihirpat1 should we fix the CMIS_DATA_MAP also which includes QSFP_DATA_MAP fields not applicable for CMIS?

Reply:
@prgeor Sure - Yes, we can fix here and at the below location as well
https://github.com/sonic-net/sonic-platform-common/blob/c6f6bed9eae562d1a045bbf62601216707969ca9/sonic_platform_base/sonic_xcvr/api/public/cmis.py#L81

Some of the unwanted fields in CMIS are
1. encoding
2. ext_rateselect_compliance
3. nominal_bit_rate

Reply:
@az-pz can you address this